include/ignition/gui/qml/Chart.qml

```diff
@@ -0,0 +1,547 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.0
+import QtCharts 2.2
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Styles 1.4
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+
+Rectangle {
+    id: main
+    property int chartID: -1
+    property bool multiChartsMode: false
+    color: "transparent"
+    signal subscribe(real Id, string topic, string path);
+    signal unSubscribe(real Id, string topic, string path);
+    signal componentSubscribe(string entity, string typeId, string type, string attribute, real Id);
+    signal componentUnSubscribe(string entity, string typeId, string attribute, real Id);
+    signal clicked(real Id);
+
+    function appendPoint(_fieldID, _x, _y)
```

Comment:
Can you document all these functions and properties? Use the `/** */` notation like here:

https://github.com/ignitionrobotics/ign-gui/blob/58cedf4c96b4baf46a0c5c79234faf9e3dc4e924/src/plugins/world_control/WorldControl.qml#L50-L53